src/actions/userActions.js

```diff
@@ -25,6 +25,19 @@ export const login = user =>
     }
   };
 
+export const signUp = user =>
+  async (dispatch) => {
+    try {
+      const response = await userApi.signUp({ user });
```

Comment:
here you can just do `const { user } = ...`

Reply:
I couldn't  because it intends to rename the existing user var

Reply:
oh ok, you can always do sth like `const { user: newUser } = ...` but in this case I think it's not worth it. That is useful when you have long paths to variables that are named the same.